internal/pritunl/user.go

```diff
@@ -25,6 +25,7 @@ type User struct {
 	Gravatar        bool                     `json:"gravatar,omitempty"`
 	OtpAuth         bool                     `json:"otp_auth,omitempty"`
 	DeviceAuth      bool                     `json:"device_auth,omitempty"`
+	DynamicFirewall bool                     `json:"dynamic_firewall,omitempty"`
```

Comment:
Do you really need this property for the `pritunl_user` resource?

Reply:
@QuentinBtd ^ minor changes requested

Reply:
You saw it right, it's a mechanical addition on my part.
I checked the API calls to Pritunl, we can also remove `DeviceAuth` and `OtpAuth` from user resource